Makefile

```diff
@@ -208,7 +208,7 @@ uninstall: generate_deployment
 # leave workspaces in a hanging state if we add finalizers.
 	$(K8S_CLI) delete devworkspaces.workspace.devfile.io --all-namespaces --all --wait || true
 	$(K8S_CLI) delete devworkspacetemplates.workspace.devfile.io --all-namespaces --all || true
-	$(K8S_CLI) delete workspaceroutings.controller.devfile.io --all-namespaces --all --wait || true
+	$(K8S_CLI) delete devworkspaceroutings.controller.devfile.io --all-namespaces --all --wait || true
```

Comment:
I'm not sure what we should do here -- with this change there seems to be no automated way to delete `workspaceroutings` from the cluster. I suppose it's not really a problem since we generally require a clean uninstall + reinstall, but it's worth noting that the actual process would be
```
git checkout <before this commit>
make uninstall
git checkout <after this commit>
make install
```

Reply:
> we generally require a clean uninstall + reinstall

Yes and not. I don't think Che operator is going to reinstall everything for upgrading from v0.1.1 to v0.2.0. 
I think that we agreed that at this point it's OK if we just leave workspaceroutings on the cluster... But it makes sense to keep removing of workspacerouting in out `make uninstall` and on  Che Operator side for a couple of releases.

Maybe then it's a good time to remove components CRDs if we don't need them anymore.

Reply:
make uninstall
make install
the only safe option =(
Is Che Operator supposed to drop every DevWorkspace instance before upgrade?...

Reply:
Alternative: we assume that workspacerouting and devworkspacerouting has the same stuff to finalize.
So, Che Operator updated DWO, update WorkspaceRouting to has [] as finalizers, remove all WorkspaceRoutings CRs, remove CRD.

Reply:
I think at this point in time we should not worry about leaving garbage behind in the Che-related usecases -  we're in the alpha's in Che so this kind of stuff is expected IMHO and can be simply documented in the release notes or somewhere.

I think the situation is different for webterminal though where I assume we need to make sure that the upgrade of the operator is not going to cause any stuck devworkspaces...